Dockerfile

```diff
@@ -28,12 +29,8 @@ RUN set -xe \
       xsl \
       zip \
       ldap \
-    && pecl install apcu pcov xdebug \
-    && curl -Ls https://github.com/websupport-sk/pecl-memcache/archive/refs/tags/8.2.tar.gz | tar xz -C / \
```

Comment:
This memcache change seems unrelated to the ImageMagick change and as such I think this would be best contributed as a separate pull request to avoid introducing side effects with these changes.

Reply:
Fixed in https://github.com/artefactual/atom/pull/2194/commits/2179fcf830ede4adbd32d4556e7a85aa7b1cbf45 (and https://github.com/artefactual/atom/pull/2194/commits/138b8f70c4fcd97f14f2bb7cf62402e8ac6d192a)